src/templates/settings.twig

```diff
@@ -0,0 +1,85 @@
+{#
```

Comment:
remove it.

Reply:
please check once. I didn't find @@ -0,0 +1,85 @@ line in setting.twig